flocker/provision/_install.py

```diff
@@ -149,6 +149,46 @@ def task_disable_selinux(distribution):
         raise NotImplementedError()
 
 
+def task_install_control_certificates(ca_cert, control_cert, control_key):
+    """
+    Install certificates and private key required by the control service.
+
+    :param FilePath ca_cert: Path to CA certificate on local machine.
+    :param FilePath control_cert: Path to control service certificate on
+        local machine.
+    :param FilePath control_key: Path to control service private key
+        local machine.
+    """
+    return sequence([
+        run('mkdir -p /etc/flocker'),
+        run('chmod 0700 /etc/flocker'),
```

Comment:
mnemonics are more typing but easier reading, `chmod u=rwX,g=,o= foo`, probably generally good to prefer that form

:question: